templates/careers/departments.html

```diff
@@ -0,0 +1,108 @@
+      <h2>Support Engineering</h2>
+      <p>Getting it done right and solving tough technical challenges are the best feelings in the world. Whether it's running complex technical infrastructure or challenging projects, your consistent focus on technology, organisation and teamwork means you're a force for good in the battle against entropy. Time zones are easy and technical customers are your favourite.</p>
+    </div>
+    <div class="col-4 u-sv3">
+      <h2>Finance</h2>
+      <p>Finance at Canonical isn't just about numbers. It's about connecting with people across the business, helping them to understand the value they bring, as well as promoting the commercial focus that will grow our brand.</p>
+      <a href="/careers/all?filter=Finance">Check out open Finance roles</a>
+    </div>
+  </div>
+
+  <div class="p-strip is-shallow u-no-padding--top">
+    <hr class="is-fixed-width">
+    <div class="row">
+      <div class="col-4 u-sv3">
+        <h2>Web and Design</h2>
+        <p>Crafts, defines and develops the user-experience in the form of a design system that supports sophisticated technical specialists across a wide range of industries. We design and build all web interfaces, complex web apps and support Ubuntu itself. With a key focus on accessibility and performance.</p>
+        <a href="/careers/all?filter=Web+%26+Design">Check out open Web and Design roles</a>
+      </div>
+      <div class="col-4 u-sv3">
+        <h2>Marketing</h2>
+        <p>We deliver the right technology message to a highly competent audience, at exactly the right moment. Success depends on having good instincts for tech trends and topics and a natural inclination to leadership.</p>
+        <a href="/careers/all?filter=Marketing">Check out open Marketing roles</a>
+      </div>
+      <div class="col-4 u-sv3">
+        <h2>Project Management</h2>
+        <p>The EPMO practice at Canonical is a central capability spearheading Canonical's Execution Excellence Strategy. EPMO project and program managers lead the delivery of a wide variety of projects ranging from commercial customer projects, like enablement of Ubuntu on new hardware and the delivery of complex private clouds, to internal Canonical strategic projects and programs. We are looking for continuous learners who are passionate about project management and technology.</p>
+        <a href="/careers/all?filter=Project+Management">Check out open Project Management roles</a>
+      </div>
+    </div>
+  </div>
+  
+  <div  class="p-strip is-shallow u-no-padding--top">
+    <hr class="is-fixed-width">
+    <div class="row">
+      <div class="col-4 u-sv3">
+        <h2>Sales</h2>
+        <p>You build long term relationships and you always promote the approach that is in the customers best interests. You are interested in the state of the art, you love to represent a company that knows how to transform customer operations.</p>
+        <a href="/careers/all?filter=Sales">Check out open Sales roles</a>
```

Comment:
Are we adding `>` on links? 
```suggestion
        <a href="/careers/all?filter=Sales">Check out open Sales roles&nbsp;&rsaquo;</a>
```

Reply:
We aren't using them for these pages 🙂 I'll update the doc!